### DISCUSSION TITLE
DBeaver UI Layout Suggestion

### DISCUSSION THREAD

**Discussion by katarot** [dbeaver/dbeaver#35215] [Ideas]
Hello, 
Is there a way to modify the UI layout like it is shown in the following screenshot?
Notice the bottom left of the screen. 
![dbeaver-ui-layout-suggestion](https://github.com/user-attachments/assets/70c20bcf-6755-43f2-987d-48a1a2e3e717)

Is it possible to make the results view window wider without affecting the code editor window width size?
If this is not possible, then this is an idea.

**Comment by E1izabeth** [comment]
Here is an issue with similar request #35236

**Reply by TB-HunBoka** [reply]
> Here is an issue with similar request #35236

I am afraid the linked ticket is about a different issue/question.
This question/suggestion is about the results tab being able to reach both side of the screen, and the database window could be above the results tab. Which would require to move the left side panel in the same area as the SQL window. 
Like this:
![image](https://github.com/user-attachments/assets/3054d1a8-81ea-43ca-80b0-9a2b9aed7327)
However, I personally prefer the current layout.

However, I think a possible good solution for @katarot is to minimize the database window
![image](https://github.com/user-attachments/assets/d4d5bda8-d654-4419-b53c-f735ccdc455c)
Then clicking on the database on the sidebar will open it in a popup window:
![image](https://github.com/user-attachments/assets/82f044d4-5ecf-43a1-8ecb-0d6c6a8262e5)

Or if you need to see that window all the time, simple detach the database navigator and move it away. It will stay visible.
![image](https://github.com/user-attachments/assets/b7a069b2-128e-43d9-b770-f663b3967e41)
